src/style.scss

```
@@ -13,6 +13,18 @@
   text-align: left;
 }
 
+:host {
+  /* Custom property to manipulate the height of auro-select element */
+  --trigger-height: 40px;
+
+  auro-dropdown {
+    &::part(trigger) {
+      min-height: var(--trigger-height);
+      max-height: var(--trigger-height);
+    }
+  }
+}
+
```

You don't need to wrap this in a `:host` selector. This will work. 

```suggestion
auro-dropdown {
  --trigger-height: 40px;

  &::part(trigger) {
    min-height: var(--trigger-height);
    max-height: var(--trigger-height);
  }
}

```

---

Fixed.